src/components/Banner/Banner.tsx

```diff
@@ -16,7 +16,7 @@ export type BannerProps = {
   /**
    * CSS class names that can be appended to the component.
    */
-  className?: ReactNode;
```

Comment:
This is the only component using the `ReactNode` type for the `className` prop; all the other components use `string`. I think this is my mistake and I don't remember how it got in there.